packages/app-store/shimmervideo/lib/getShimmerAppKeys.ts

```diff
@@ -8,6 +8,6 @@ const shimmerAppKeysSchema = z.object({
 });
 
 export const getShimmerAppKeys = async () => {
-  const appKeys = await getAppKeysFromSlug("shimmer-video");
+  const appKeys = await getAppKeysFromSlug("shimmervideo");
```

Comment:
Lines up with the slug in the config file

Reply:
We should read this from config